test/test-register.jl

```diff
@@ -0,0 +1,2044 @@
+using Dates
+using LinearAlgebra: dot, det, norm
+using IceFloeTracker: register, imrotate_bin_counterclockwise_radians
+
+@testset "rotation" begin
+    @testset "individual rotations" begin
+        unit_vector(θ) = [cos(θ); sin(θ)]
+        oriented_angle_between_vectors(a, b) = atan(det(hcat(a, b)), dot(a, b))
```

Comment:
Didn't we have this change in already?
```suggestion
        oriented_angle_between_vectors(u, v) = atan(det(hcat(u, v)), dot(u, v))
```